client/webserver/site/src/js/doc.js

```diff
@@ -102,6 +102,11 @@ export default class Doc {
     return el.classList.contains('d-hide')
   }
 
+  /* isDisplayed returns true if the specified element is not hidden */
+  static isDisplayed (el) {
+    return !el.classList.contains('d-hide')
```

Comment:
Wouldn't it be tighter to call `!isHidden` instead of defining `isDisplayed`?

Reply:
The two are equivalent, but I wasn't crazy about `!isHidden` where I needed it. I thought `isDisplayed` reads better in some contexts.